db/fixtures/miq_product_features.yml

```diff
@@ -5123,3 +5123,36 @@
     :description: Delete Arbitration Settings
     :feature_type: admin
     :identifier: delete_arbitration_setting
+
+# Blueprints
+- :name: Blueprints
+  :description: All Blueprints Access Rules
+  :feature_type:
```

Comment:
@h-kataria thanks! I will update accordingly. Do you have any advice for the `:feature_type`?